packages/visualizations/app/styles/navi-visualizations/visualizations/table.less

```diff
@@ -18,17 +18,19 @@
   overflow-y: auto;
   overflow-x: hidden;
 
-  &__row-info {
+  &__info-header {
+    align-items: center;
     display: flex;
     font-size: @font-size-mid;
-    justify-content: flex-end;
-    padding: 3px 0;
-    margin-right: 12px;
+    flex-flow: row-reverse;
+    justify-content: space-between;
+    margin: @row-margin;
+    padding: 6px 0 0;
+  }
```

Comment:
Have you tried this in firefox? I Know it can be a little fiddly with some flex settings (though I don't see anything here that concerns me)

Reply:
Yep. Chrome, FF, & Safari look good.